test/CMakeLists.txt

```diff
@@ -145,24 +169,37 @@ endmacro()
 
 set(_regexp_dpcpp_backend_required "(xpu_api/algorithms|xpu_api/functional|xpu_api/numerics)")
 set(_regexp_switch_off_checked_it  "(test/general/header_order_ranges|test/parallel_api/algorithm/alg.sorting/alg.min.max|test/parallel_api/ranges|test/xpu_api/random/device_tests|test/xpu_api/random/interface_tests|test/xpu_api/random/statistics_tests|test/parallel_api/numeric/numeric.ops/transform_scan)")
+set(_skip_regex_for_not_std "(std/)")
 file(GLOB_RECURSE UNIT_TESTS "*.pass.cpp")
 foreach (_file IN LISTS UNIT_TESTS)
 
+    if (_file MATCHES "${_skip_regex_for_not_std}")
+        continue()
+    endif()
+
     if (_file MATCHES "${_regexp_dpcpp_backend_required}")
         if (ONEDPL_BACKEND MATCHES "^(dpcpp|dpcpp_only)$")
             # Switch off checked iterators
-            onedpl_add_test(${_file} true)
+            onedpl_add_test(${_file} true false)
         else()
             message(STATUS "Skip test ${_file} as it requires DPC++ backend")
         endif()
     else()
         if (_file MATCHES "${_regexp_switch_off_checked_it}")
             # Switch off checked iterators
-            onedpl_add_test(${_file} true)
+            onedpl_add_test(${_file} true false)
         else()
             # Do not switch off checked iterators
-            onedpl_add_test(${_file} false)
+            onedpl_add_test(${_file} false false)
         endif()
     endif()
 
 endforeach()
+
+file(GLOB_RECURSE UNIT_TESTS "std/*.pass.cpp")
+foreach (_file IN LISTS UNIT_TESTS)
+
```

Comment:
Extra newlines at the beginning and end of this `foreach` block.

Reply:
Fixed.